sequence_processing_pipeline/Pipeline.py

```diff
@@ -12,11 +12,79 @@
 import sample_sheet
 import pandas as pd
 from collections import defaultdict
+from datetime import datetime
+import pytz
+from xml.etree import ElementTree as ET
 
 
 logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
 
 
+class InstrumentUtils():
+    types = {'A': 'NovaSeq 6000', 'D': 'HiSeq 2500', 'FS': 'iSeq',
+             'K': 'HiSeq 4000', 'LH': 'NovaSeq X Plus', 'M': 'MiSeq',
+             'MN': 'MiniSeq',
+             # SN – RapidRun which is HiSeq 2500
+             'SN': 'RapidRun'}
+
+    @staticmethod
+    def get_instrument_type(run_id):
+        tmp = run_id.split('_')
+
+        # remove digits from substring to get instrument code.
+        code = ''.join([i for i in tmp[1] if not i.isdigit()])
+        try:
+            return InstrumentUtils.types[code]
+        except KeyError:
+            raise ValueError(f"Instrument code '{code}' is of unknown type")
+
+    @staticmethod
```

Comment:
same question ... why staticmethod vs. a regular method?

Reply:
Just realized that all are static methods, if that's the case why have an object for them vs. regular methods?

Reply:
see above.